data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/parser/LogstashConfigConverterTest.java

```diff
@@ -0,0 +1,29 @@
+package org.opensearch.dataprepper.logstash.parser;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+class LogstashConfigConverterTest {
```

Comment:
This test will have some overlap with the integration tests. If we aren't going to mock the internal classes, I think I'd prefer that we remove this test and only have the integration test to cover it.

Reply:
I think we can only have an integration test for this.

Reply:
Then let's just remove this test and the input file.